bot.py

```diff
@@ -17,43 +19,56 @@ def button_message(message):
     markup.add(item1)
     bot.send_message(message.chat.id,'Click the button below :)',reply_markup=markup)
 
+def Button(token, call):
+    keyboard_button = types.InlineKeyboardButton(text = token, callback_data= call)
+    return keyboard_button
+
+
 @bot.message_handler(content_types=['text'])
 def add_coin_f(message):
     if message.text == "Realtime rates":
```

Comment:
Здесь чтобы от лишнего отступа избавиться, можно сделать так
```suggestion
    if message.text != "Realtime rates":
        return
```

Reply:
Тут кстати тоже в названии функции одна буква f, без чтения её тела не понятно, что она делает, лучше не сокращать